Gormsson/Source/GattCharacteristic/TemperatureMeasurement.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  TemperatureMeasurement.swift
+//  Gormsson
+//
+//  Created by Loïc GRIFFIE on 16/04/2019.
+//  Copyright © 2019 Loïc GRIFFIE. All rights reserved.
+//
+
+import CoreBluetooth
+
+internal final class TemperatureMeasurement: CharacteristicProtocol {
+    public var uuid: CBUUID {
+        return CBUUID(string: "2A1C")
+    }
+
+    public var service: GattService {
+        return .healthThermometer
+    }
+
+    public var format: DataInitializable.Type {
+        return TemperatureMeasurementType.self
+    }
+}
+
+public enum TemperatureMeasurementTypeEnum: UInt8, CustomStringConvertible {
+    case armpit = 1
+    case body
+    case ear
+    case finger
+    case gastroIntestinalTract
+    case mouth
+    case rectum
+    case toe
+    case tympanum
+
+    case unknown
```

Comment:
unsupported should be better (or at least I start to use this in the coming PR ;)